components/post_view/post_info/post_info.jsx

```diff
@@ -224,6 +224,7 @@ export default class PostInfo extends React.PureComponent {
                             <button
                                 className='reacticon__container color--link style--none'
                                 onClick={this.toggleEmojiPicker}
+                                title={Utils.localizeMessage('post_info.tooltip.add_reactions', 'Add Reactions')}
```

Comment:
Maybe "Add reactions" must be singular instead of plural.

Reply:
The ticket specified "Add Reactions" plural, but I agree you might have a point. @esethna which one should it be?

Reply:
Thanks, good catch. Lets go with "reaction", singular